### PR TITLE
Support for topojson (in example)

### PR DESCRIPTION
**Pull request by korczis**
Any comments/suggestions?

cc: @mourner

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -24,6 +24,7 @@
     "faucet": "0.0.1",
     "istanbul": "^0.4.0",
     "tape": "^4.2.2",
+    "topojson": "^1.6.19",
```
Comment (mourner) [inline package.json]: Do we need TopoJSON as a dependency if we're including a CDN version?

Comment (korczis) [inline package.json]: Nope, I will fix this in second. Sorry.

Comment (korczis) [inline package.json]: Fixed.

**Comment by mourner** [comment]
Merged in #74

**Pull request closed without merging**